\documentclass[letterpaper, 10pt]{article} %norsk

  % Packages
\usepackage[left=2cm, right=2cm, top=3cm]{geometry}
\usepackage{hyperref}
\usepackage{multicol}
\usepackage[usenames,dvipsnames]{color}
\usepackage{graphicx}
\usepackage{url}
\usepackage{longtable}
\usepackage{enumitem}

\usepackage{natbib}
\usepackage{bibentry}
\nobibliography{cv}


  % Set up hyperref
\hypersetup{
  colorlinks,
  breaklinks,
  linkcolor=NavyBlue,
  urlcolor=NavyBlue,
  pdftitle={Thies Gehrmann Curriculum Vitae},
  pdfauthor={Thies Gehrmann},
  pdfsubject={Curriculum Vitae},
  pdfcreator={Thies Gehrmann},
  pdfproducer={Thies Gehrmann},
}

  % Geometry 
\textheight 23 cm
\textwidth 16 cm
\oddsidemargin 0 cm
\evensidemargin 0 cm

  % Create a function for quoting
\renewcommand{\quote}[1]{
  ``\emph{#1}''
}

  % Make lists without bullets
\renewenvironment{itemize}{
  \begin{list}{$\cdot$}{
    \setlength{\itemsep}{0pt}
    \setlength{\topsep}{7pt}
    \setlength{\leftmargin}{0pt}
    \setlength{\itemindent}{10pt}
  }
}{
  \end{list}
}

 % A new section header
\newcommand{\nsection}[1]{
    \section*{\color{MidnightBlue}#1}
  \vspace{-1mm}
}

  % A non italicized item in a section
\newcommand{\entryni}[2]{
  \noindent
  \begin{tabular}[t]{p{0.2\textwidth}|p{0.8\textwidth}}
    \textbf{\color{BrickRed}#1} & {#2} \\
  \end{tabular}
  \vspace{-2mm}

}

 % An italicized item in a section
\newcommand{\entry}[3]{
  \noindent
  \begin{tabular}[t]{p{0.2\textwidth}|p{0.8\textwidth}}
    \textbf{\color{BrickRed}#1} & {#2 \textit{#3}} \\
  \end{tabular}
  \vspace{-2mm}
}


  % No paragraph indent
\setlength\parindent{10pt}

  % Data definitions
\def\doctitle{\color{BrickRed}Curriculum Vit\ae}
\def\name{\color{MidnightBlue}Thies Gehrmann}
\def\maxdeg{\color{Black}PhD}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                    END LATEX PREAMBLE                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

\small

\pagestyle{plain}

\label{top}

  % Title
{\huge {\textbf \name} {\footnotesize $\,$ \maxdeg} $\;$ {\textbf \doctitle} }
\vspace{0.1cm}
\hrule
\vspace{0.5cm}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                    CONTACT & CONTENT INFORMATION                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{minipage}[t]{0.50\textwidth}
  (email): \href{mailto:thiesgehrmann@gmail.com}{thiesgehrmann@gmail.com} \\
  (https): \href{https://thiesgehrmann.github.io}{thiesgehrmann.github.io} \\
  (tel NL): +31 (0)640 175 860     \\
  %(tel UK): +44 (0)7517 221 445    \\
  \\
  D.O.B. April $8^{th}$ 1989 \\
  Nationality: German \\%\includegraphics[scale=0.4]{de.eps} German\\
  \hrule
  \vspace{5pt}

\end{minipage}
\hfill
\begin{minipage}[t]{0.30\textwidth}
  %\includegraphics[scale=0.4]{nl.eps}
  \textbf{Private address:} \\
  Morskade 14 \\
  2332 GB Leiden \\
  The Netherlands \\
%  \\
%  \includegraphics[scale=0.4]{nl.eps}
%  \textbf{Work address:} \\
%  EWI HB13.090 \\
%  Mekelweg 4\\
%  2628 CD Delft \\
%  The Netherlands \\
%  \\

  Last updated \today.
\end{minipage}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                      PERSONAL PROFILE                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Personal profile}
  \label{sec:persprof}

  Currently, I am a Postdoc in the department of Molecular Epidemiology at the Leiden University Medical Centre, working on translational research of healthy ageing.
  %Having been raised in Norway, among people from all over the world, I am accustomed to an international, multicultural environment.
  Bioinformatics is a constantly developing field, with continuously evolving measurements that presents a splendid opportunity to develop and apply novel methods.
  It also requires the practitioner to develop multidisciplinary skills from the statistical, computational and biological domains.
  I am excited to be part of this field, where my knowledge and competences are constantly challenged, humbled and developed.
  %I am looking for a position where challenging problems, involving many different skills and team work, allow me to contribute positively to society, and further develop my skills.
  
  
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                         EMPLOYMENT                                        %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Employment}
	\label{sec:employment}
	
	\entry{Jan 2018 - Present}
	      {Postdoc at Department of Molecular Epidemiology, Leiden University Medical Centre \newline}
	      {Bioinformatician on healthy ageing, together with Max Plank Institute of Biological Ageing. \newline}
	      
	\entry{Nov. 2016 - 2017}
	      {Postdoc at KNAW/Westerdijk Institute of Fungal Biodiversity \newline}
	      {Embedded bioinformatician. \newline}
	      
	\entry{Nov. 2012 - 2016}
	      {PhD Candidate \newline}
	      {Conducted research and performed teaching.}

%  \entry{January 2012 - \newline June 2012 }
%        {Teaching Assistant - Course Computational Molecular Biology \newline}
%        {I assisted with the course \textit{Computational Molecular Biology}, part of the Master's degree in Bioinformatics at Leiden University.
%         This involved assisting with the material, creating and grading assignments, and collecting datasets for course use.
%        }


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                        EDUCATION                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Education}
  \label{sec:education}

  \entry{2012-2018}
        {Pattern Recognition and Bioinformatics group, TU Delft, The Netherlands \newline}
        {PhD in Bioinformatics. \newline
         \textit{Bioinformatic Analysis of Genetic and Transcriptomic Variation in Fungi}.
         \newline}

  \entry{2010-2012}
        {Leiden University, Leiden, The Netherlands (In cooperation with TU Delft) \newline}
        {MSc Computer Science Track Bioinformatics. \newline
         Thesis on protein function prediction using Conditional Random Fields.
         \newline}

  \entry{2008-2010}
        {Heriot Watt University, Edinburgh, Scotland \newline}
        {BSc (Ord) Computer Science \newline
         Graduated with distinction.
         }

%  \entry{2007-2008}
%        {Napier University, Edinburgh, Scotland \newline}
%        {Certificate of Higher Education \newline
%         Completed year 1 of Computer Studies, then transferred to Heriot Watt University.
%         \newline}

%  \entry{2000-2007}
%        {International School of Stavanger, Stavanger, Norway \newline}
%        {Completed the IB and IGCSE diplomas.}





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                           SKILLS                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Skills}
  \label{sec:skills}

  \entryni{Technical Skills}
          {Python, Scala, C/C++, Java, Shell, Matlab, R\newline
           Experience with schedulers (SLURM, PDB) and version control software, (SVN, Git).\newline
           SQL, XML (and related technologies), \LaTeX. \newline
           Pipeline development and management (Snakemake, Anaconda, Jupyter Notebook). \newline
           Experienced with Unix environments (primarily Linux).\newline
           Machine learning, comparative genomics, RNA-Seq analysis and algorithm development.
           %Ability to easily use new tools and methods, and if all fails, develop his own.
           \newline}
  \entryni{Languages}
          {Fluent in Norwegian, German and English. \newline
           High level Dutch. Beginners French.
          }

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                         ACTIVITIES                                        %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\nsection{Activities}
%  \label{sec:activities}
%
%  \entry{Spring-Autumn 2012}
%        {Master's Thesis on protein function prediction using Conditional Markov Fields\newline}
%        {Changing a previous Markov Random Field model to a CRF model to demonstrate that CRFs can be used to combine several sources of information to predict protein function.
%         \newline}
%
%  \entry{Autumn-Winter 2011}
%        {Literature research project as part of Master's degree \newline}
%        {As a study of contextually aware machine learning methods which can utilize heterogeneous data, Conditional Random Fields were studied.
%         Their use in Bioinformatics was examined in several applications, including gene finding, named entity recognition and protein-protein interaction site prediction.
%        }
%
%  \entry{Spring 2010}
%        {Software development project at Heriot Watt University \newline}
%        {Part of a continuous assessment at Heriot Watt University.
%         The involved writing primitive CGI libraries in C;
%         Thies was responsible for the development and served as project manager.
%         }
%
%  \entry{Summer 2008}
%        {Unpaid learning experience at Napier University \newline}
%        {An attempt was made to develop code for the Silabs CP220x embedded ethernet controller which compiled with the open source SDCC (Small Device C Compiler) to replace proprietary web server code written for proprietary Keil compilers and libraries.
%        }




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                       APPLICATION SHOWCASE                                %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\nsection{Application Showcase}
%  \label{sec:showcase}
%
%  \entry{DT-cut}
%        {A tool and generalized toolbox for discovering significant clusters in any data.\newline}
%        {\url{https://github.com/thiesgehrmann/DTcut}
%        \newline}
%
%  \entry{ARA}
%        {A toolkit for marker and SNP analyis and quantification. Developed with Thomas Abeel and Arlin Keo.\newline}
%        {\url{https://github.com/AbeelLab/ara}
%        \newline}
%
%  \entry{Proteny}
%        {A toolbox developed with Python to perform a Synteny analysis over two genomes.\newline}
%        {\url{https://github.com/thiesgehrmann/proteny}
%        \newline}
%
%  \entry{RNA-Seq Pipeline}
%        {A pipeline developed with Python and shell tools to perform analysis on RNA-Seq data.\newline}
%        {\url{https://github.com/thiesgehrmann/delftrnaseq}
%        \newline}
%
%  \entry{CRFunc}
%        {A tool developed to integrate multiple sources of data for protein function prediction using conditional random fields.\newline}
%        {\url{https://github.com/thiesgehrmann/CRFunc}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                         REFEREES                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\nsection{Referees}
  \label{sec:referees}
%
  \entry{Upon request}{Contact Thies for referee contact details.}{}
%
%  \entry{Thomas Abeel}
%        {Assistant professor in the Delft Bioinformatics Lab \newline}
%        {\href{mailto:T.Abeel@tudelft.nl}{T.Abeel@tudelft.nl} \newline
%         Doctoral co-promotor and daily supervisor.
%         \newline}

%  \entry{Marcel Reinders}
%        {Group leader of the Delft Bioinformatics Lab \newline}
%        {\href{mailto:M.J.T.Reinders@tudelft.nl}{M.J.T.Reinders@tudelft.nl} \newline
%          Doctoral promoter and supervisor.
%         \newline}

%  \entry{Anton Sonnenberg}
%        {Leader of Mushroom Research Group, Wageningen University\newline}
%        {\href{mailto:anton.sonnenberg@wur.nl}{anton.sonnenberg@wur.nl} \newline
%         Collaborator and STW Project user committee member.
%        }
%
%  \entry{Han W\"{o}sten}
%        {Professor in Microbiology at Utrecht University\newline}
%        {\href{mailto:H.A.B.Wosten@uu.nl}{H.A.B.Wosten@uu.nl} \newline
%         Collaborator
%         \newline}

%  \entry{Dick de Ridder}
%        {Professor in Bioinformatics in the WU Plant Sciences group \newline}
%        {\href{mailto:dick.deridder@wur.nl}{dick.deridder@wur.nl}  \url{http://www.wageningenur.nl/en/Persons/prof.dr.ir.-D-Dick-de-Ridder.htm}\newline
%         Master's thesis supervisor.
%         \newline}

%  \entry{Marco Loog}
%        {Assistant professor at the TU Delft Computer Vision lab \newline}
%        {\href{mailto:m.loog@tudelft.nl}{m.loog@tudelft.nl} \url{http://visionlab.tudelft.nl/users/marco-loog}\newline
%         Master's Thesis supervisor.
%         \newline}

%  \entry{Erwin Bakker}
%        {Assistant professor at the Leiden Institute of Advanced Computer Science \newline}
%        {\href{mailto:erwin@liacs.nl}{erwin@liacs.nl}  \url{http://www.liacs.nl/~erwin/}\newline
%         Very familiar with Thies' academic performance.
%         \newline}

%  \entry{Murdoch Gabbay}
%        {Lecturer at the department of Mathematical and Computer Sciences at HWU \newline}
%        {\href{mailto:gabbay@macs.hw.ac.uk}{gabbay@macs.hw.ac.uk}  \url{http://www.macs.hw.ac.uk/~gabbay}\newline
%         He offered considerable advice and inspiration.
%        }

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                       EDUCATIONAL ACTIVITIES                              %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Teaching}
  \label{sec:teaching}

  \entry{MSc Courses}{\textit{Functional Genomics and Systems Biolog}y}
        { \newline {\normalfont Lecturer and teaching assistant. Fall 2014, 2015, and 2016 at TU Delft \newline} }
        
  \entry{}{\textit{Computational Molecular Biology}}
		{ \newline {\normalfont Teaching assistant. Spring 2012 at Leiden University } \newline }
		
  \entry{BSc Courses}{\textit{Life Science and Technology Bioinformatics}}
        { \newline { \normalfont Teaching assistant. Spring 2016 at TU Delft.} \newline} 
        
  \entry{}{\textit{Genome Scale Data Analysis}}
		{ \newline { \normalfont Lecturer and teaching assistant. Fall 2014 at TU Delft } \newline}
		
  \entry{External Courses}{\textit{Quantitative biology summer school}}
        { \newline { \normalfont Lectured on computational aspects of synteny and alternative splicing in fungi. Summer 2015 at Utrecht University.}}

\nsection{Student Supervision}
  \label{sec:supervision}
  \entry{Jet Beekwilder}{B.sc Student 2017-2018\newline}
        {Comparison of DNA extraction methods from fungi for Nanopore sequencing.\newline}
        
  \entry{Andr\'{e} Vollering}{M.Sc Student 2014-2015 \newline}
        {Heterosis: Finding Associated Genomic Regions. \newline}
        
  \entry{Valerie Pourquie}{B.Sc Student 2015 \newline}
        {Conservation of polarization proteins in yeast and fungi. \newline}
        
  \entry{Dimitris Palachanis}{M.Sc Student 2013-2014 \newline}
        {Using the Multiple Instance Learning framework to address differential regulation.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                       OTHER SKILLS/HOBBIES                                %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Other activities/skills/information}
  \label{sec:oinfo}

  \bibliographystyle{plainnat}
  \entryni{Awards}
          {ECCB 2016 Student Symposium 'Best Oral Presentation' award.\newline
           TU Delft Graduate School 'Best poster presentation' award 2013.\newline
           Scott Logic Computer Science Prize 2010. \newline} 
  \entryni{Reviewing}
          {Nucleic Acid Research (2016)\newline}
  \entryni{Volunteering}
          {International Student Network (Leiden) 2013-2016. Stage construction and management.\newline
           Dutch Red Cross (Den Haag) 2014-2015. First Aid (EHBO) at events and refugee centers.}
%  \entryni{Professional Courses followed}
%          {2014 - NBIC: Comparative Genomics.\newline
%           2013 - NBIC: Advanced NGS - RNA-Seq.\newline
%           2014 - CSHL: Statistical Methods in Functional Genomics\newline
%          }
%  \entryni{Transferable skills}
%           2013 - TUD: Art of presenting science.\newline
%           2013 - TUD: Teaching and active learning.\newline
%           2013 - TUD: Coaching individual students.\newline
%           CSHL: Statistical Methods in Functional Genomics.\newline
%          }
%  \entryni{Special Interests}
 %         {Theatre acting, Sailing, Cycling, Climbing/Bouldering, Swimming, Skiing and Hiking.
%          }

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                         PUBLICATIONS                                      %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\nsection{Publications}
\label{sec:publications}

\entryni{Journal \newline publications}
{
	
	\begin{enumerate}[label={[\arabic*]},nosep]
		%\item \bibentry{gehrmann2015isoform}
		\item \bibentry{gehrmann2018nucleus} \newline
		\item \bibentry{manson2017mycobacterium} \newline
		\item \bibentry{pelkmans2017transcription} \newline
		\item \bibentry{gehrmann2016isoform} \newline
		\item \bibentry{pelkmans2016c2h2} \newline
		\item \bibentry{gehrmann2015proteny}\vspace*{-\baselineskip}
	\end{enumerate}
}

\entryni{Conference \newline publications}
{\begin{enumerate}[label={[\arabic*]},nosep]
		\addtocounter{enumi}{6}
		\item \bibentry{gehrmann2013conditional} \vspace*{-\baselineskip}
	\end{enumerate}
}

\entryni{Under Review}
{
	\begin{enumerate}[label={[\arabic*]},nosep]
		\addtocounter{enumi}{7}
		\item \bibentry{gehrmann2017variants}  (\textit{Under review in Scientific Reports.}) \newline
		\item \bibentry{diepeveen2017patterns} (\textit{Under review in Molecular Biology and Evolution}) \newline
		\item {Duong Vu, Michel de Vries 1, Thies Gehrmann, Benjamin Stielow, Ursula Eberhardt, Abdullah Al-Hatmi, Ewald Z. Groenewald, Marizeth
			Groenewald, Gianluigi Cardinali, Teun Boekhout, Pedro W. Crous, Vincent Robert, and Gerard J.M. Verkleij. Large-scale analysis of filamentous fungal DNA barcodes reveals thresholds for species and higher taxon delimitation. (\textit{Under review in Studies in Mycology}) \newline}
		\item {Vladimiro Guarnaccia, Thies Gehrmann, Duong Vu, Vincent Robert, Ewald Z. Groenewald, Pedro W. Crous.  \textit{Phyllosticta citricarpa} and sister species of global importance to Citrus.  (\textit{Under review in Molecular Plant Pathology}) }
		\vspace*{-\baselineskip}
	\end{enumerate}
}

\entryni{In Progress}
{
	\begin{enumerate}[label={[\arabic*]},nosep]
		\addtocounter{enumi}{10}
		\item {Lizel Mostert, Thies Gehrmann, Duong Vu, Vincent Robert. Genomic analysis of the Grape vine pathogen \textit{Phaeoacremonium} in South African vineyards.  (\textit{In progress}) } \newline
		\item {Annick Lang, Thies Gehrmann, Nils Cronberg. Extreme gender dimorphism and its impact on the genetic diversity of \textit{Dicranum scoparium} mosses. (\textit{In progress}) } \newline
		\item {Differentiating the expression of a watermarked pathway duplication in an industrial fungus. (\textit{In progress}) } \newline
		\item {Conservation of the regulation of a core metabolic pathway across fungal species. (\textit{In progress}) }
		\vspace*{-\baselineskip}
	\end{enumerate}
}

\nobibliography{cv}
\end{document}

